src/main/java/com/appbuddy/buddypasswordmanager/gui/HelpGUI.java

```diff
@@ -18,48 +19,51 @@
  */
 
 public class HelpGUI {
-	JFrame frame;
-	JPanel panel;
-	JLabel label;
-	JTextArea textArea;
-	JScrollPane scroll;
-	JButton button;
-
-	public HelpGUI() {
-		frame = new JFrame("Help");
-		frame.setSize(350, 370);
-		frame.setVisible(true);
-		frame.getContentPane();
-
-		label = new JLabel("About this Program");
-
-		textArea = new JTextArea(5, 20);
-		textArea.setText("This is the help section");
-		textArea.setFont(new Font("Comic Sans", Font.BOLD, 20));
-		textArea.setWrapStyleWord(true);
-		textArea.setLineWrap(true);
-		textArea.setEditable(false);
-
-		scroll = new JScrollPane(textArea);
-
-		frame.add(scroll);
+    private static final Logger LOGGER = Logger.getLogger(HelpGUI.class.getName());
+    private JFrame frame;
+    private JPanel panel;
+    private JLabel label;
+    private JTextArea textArea;
+    private JScrollPane scroll;
+    private JButton button;
 
-		scroll.setBounds(0, 0, 350, 370);
+    public HelpGUI() {
+        this.frame = new JFrame("Help");
+        this.label = new JLabel("About this Program");
+        this.textArea = new JTextArea(5, 20);
+        this.scroll = new JScrollPane(textArea);
+        this.button = new JButton("Button");
+        this.initFrame();
+        this.initTextArea();
+        this.frame.add(scroll);
+        this.scroll.setBounds(0, 0, 350, 370);
```

Comment:
Is there no way to set relative sizing here? Hard numbers like this makes me raise an eyebrow

Reply:
Sorry, just saw this message. I’m sure there is, but I didn’t look into changing that for this request. It’s still the same size as it was originally.